python/GangaDirac/Lib/Server/DiracCommands.py

```diff
@@ -58,6 +58,11 @@ def getReplicas(lfns):
     output(dirac.getReplicas(lfns, active=True, preferDisk = True))
 
 
+def getAccessURL(lfn,SE):
+    ''' Return the access URL for the given LFN and storage element '''
+    output(dirac.getAccessURL(lfn,SE))
```

Comment:
Stupid question: The docstring says "Return", but this function does not return anything?